docs/_news/news06.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: news
+title: New for Codewind 0.6.0
+description: New for Codewind 0.6.0
+keywords: news, new, updates, update, version
+duration: 1 minute
+permalink: news06
+---
+
+## Codewind 0.6.0
+Thursday 21 November 2019
+
+#### Codewind workspace updates
+**VS Code**
+- You no longer need to place projects in the `codewind-workspace`.
+- Do not create projects in the `~/codewind-data/` or `C:\codewind-data` directories. User configuration files are now kept in `~/.codewind/` or `C:\Users\<user>\.codewind\`.
+- If you want to move an existing project, remove it from Codewind without deleting the files. Then, run the `Add Existing Project` command to reintroduce it to its new location.
+- In VS Code, to skip the **Select a parent directory** step when you create a project, use the `codewind.alwaysCreateProjectsInWorkspace` setting.
+- Project information is now kept in the `codewind_cw-workspace` Docker volume. If this volume is deleted, your projects are deleted from Codewind. However, they still exist on your disk.
+- Project files are synchronized whenever a file changes in this Docker volume. Previously, Codewind did a Docker mount of the entire `codewind-workspace`.
+**Eclipse**
+The information in the **VS Code** section also applies to Eclipse but with these exceptions:
+- When you move a project, to add an existing project to Eclipse, right-click **Local** in the Codewind explorer view and select **Add Existing Project**.
+- When you create projects, create the project in the Eclipse workspace. Then, enter a different location if you want.
+
+#### Appsody
+- Appsody works on Che.
+
+#### Che
+- The file, **Setting up OpenShift on IKS for Codewind**, is removed because these steps are no longer needed. The instructions to set up Codewind on OpenShift on IKS are the same as a regular OpenShift installation.
+
+#### Codewind OpenAPI Tools for VS Code
+- Codewind 0.6.0 introduces improved support for Maven projects after code generation. The `.pom` file is merged, and you can access the APIs without manual steps.
```

Comment:
@keithchong will add his review comments to this line, as there are some manual steps (although many were fixed).

Reply:
Something like this:

- Codewind 0.6.0 introduces usability improvements for Maven projects after code generation. If generating into the project root folder for the first time, the existing `.pom` file is merged with the one from the generator so that the Maven build will build successfully. In some cases, there might be manual steps to expose the API endpoints because the OpenAPI tools does not override existing project configurations.